### ISSUE TITLE
Inference script + license? Looks very promising!!

### ISSUE THREAD

**Issue by ThibaultGROUEIX** [CesareDavidePace/poseidon#3]
Hi,

I would love to try this method.
I was able to install the environment without problems. 

I have three questions : 

1/3 Can you include a script to run the model on a new video / frame?
2/3 Which weights should be downloaded for VIT_POSE?
<img width="928" alt="Image" src="https://github.com/user-attachments/assets/aba4e0a9-b298-4010-9f09-4b244b6d8f4e" />
3/3 what is the licence?

Thank you,
Thibault

**Comment by CesareDavidePace** [comment]
Hi Thibault,

Glad to hear you got the environment set up without any issues!

1️⃣ A script for running the model on a new video/frame will be available soon. Stay tuned!
2️⃣ You should download the weights under the classic decoder type for ViT_POSE.
3️⃣ I have just added the Apache 2.0 license to the repository.

Let me know if you have any other questions!

Best,
Cesare

**Comment by ThibaultGROUEIX** [comment]
Awesome, thanks! 
2/ I did not understand. Which one is the classic decoder type for ViT_POSE?

**Comment by CesareDavidePace** [comment]
The classic decode type consists of two deconvolution blocks, where each block includes a deconvolution layer followed by Batch Normalization and a ReLU activation.

**Comment by ThibaultGROUEIX** [comment]
I have not dived into the detailed of architecture yet.  I am trying to gauge performance of the model first. 

Which model, out of the 9 options in the screenshot, is the one that I should use to run Poseidon inference?

Best,
Thibault

**Comment by CesareDavidePace** [comment]
To make an inference you do not have to take these weights because they are the weights of the pre-trained models that I used to train Poseidon and they are used to replicate the results obtained. The weights of the model to make inferences have not yet been published but will be soon.

**Comment by ThibaultGROUEIX** [comment]
Thanks for the clarification @CesareDavidePace !  Do you have an ETA in mind?

**Comment by CesareDavidePace** [comment]
I think in a few weeks it will be online. Thank you for your interest!
